### DISCUSSION TITLE
Hi!

### DISCUSSION THREAD

**Discussion by RedySGOLF** [Compliance-Resource-Pack/Resource-Pack-32x#25] [Q&A]
Hello, dear developers, I would like to know when the release of version 1.19.2 will be?

**Comment by Pomi108** [comment]
.2 support was ditched for the last release since that's how it's always been – Faithful's only ever supported the latest minor version of any given update. Due to high demand however, 1.19.2 support is going to be restored in the next release.

**Reply by RedySGOLF** [reply]
Thank you for the answer, I'm looking forward to it
